packages/api/src/hooks/p2p/useAdvertiserPaymentMethods.ts

```diff
@@ -0,0 +1,70 @@
+import { useCallback, useMemo } from 'react';
+import useInvalidateQuery from '../../useInvalidateQuery';
+import useAuthorize from '../useAuthorize';
+import useQuery from '../../useQuery';
+import useMutation from '../../useMutation';
+
+type TPayloads = NonNullable<
+    NonNullable<Parameters<ReturnType<typeof useMutation<'p2p_advertiser_payment_methods'>>['mutate']>[0]>['payload']
+>;
+type TCreatePayload = NonNullable<TPayloads['create']>[0];
+type TUpdatePayload = NonNullable<TPayloads['update']>[0];
+
+const type_to_icon_mapper = {
+    bank: 'IcCashierBankTransfer',
+    other: 'IcCashierOther',
+    ewallet: 'IcCashierEwallet',
```

Comment:
Question: These keys are we comparing from the API ? If we are, pls type this from the API 😸

Reply:
Thanks for your revision, since we don't need this I will remove this